js/components/thumbnailContainer.js

```diff
@@ -107,6 +109,18 @@ var ThumbnailContainer = React.createClass({
     }
   },
 
+  getClientWidth: function(element) {
```

Comment:
I think these might be a bit generic for thumbnailcontainer. Would it be better to move these to utils?

Reply:
Sounds good, will do.